ember/app/components/settings-panels/choose-club.js

```diff
@@ -28,7 +28,13 @@ export default Ember.Component.extend({
 
   saveTask: task(function * () {
     let club = this.get('club');
-    let json = { clubId: club.id };
+    let json;
+
+    if (club.id) {
+      json = { clubId: club.id };
+    } else {
+      json = { clubId: null };
+    }
 
```

Comment:
Can we simplify to `clubId: club.id || null` without the condition?

Reply:
Yes, fixed in commit 9ef16c3b170ed027f0180af85f5ccf0034dec81f